flask_dynamo/manager.py

```diff
@@ -42,8 +42,10 @@ def wait_not_exists(self, table_name):
         self._wait(table_name, 'table_not_exists')
 
     def create_all(self, wait=False):
+        tables_name_list = [table.name for table in self._connection.tables.all()]
         for table in self._table_config:
-            self._connection.create_table(**table)
+            if table['TableName'] not in tables_name_list:
+                self._connection.create_table(**table)
         if wait:
```

Comment:
my fault, I should have explained a little better...

`    def create_all(self, wait=False):
        tables_name_list = [table.name for table in self._connection.tables.all()]
        for table in self._table_config:
            if table['TableName'] not in tables_name_list:
                self._connection.create_table(**table)
                if wait:
                    self.wait_exists(table['TableName'])`

Here is the snippet as i imagined it... I should have said "You can just tab over these 2 statements, so that you only hit them when you create a table..."